Source/ProceduralFairings/FairingBase.cs

```diff
@@ -714,6 +731,19 @@ public void ShowHideInterstageNodes()
             }
         }
 
+        public void UpdateOpen()
+        {
+            if (openFairing && HighLogic.LoadedSceneIsEditor)
+            {
+                editorOpenOffset.x = OffsetValueX;
+                sides.ForEach(side => side.SetOffset(side.meshPos + editorOpenOffset));
+            }
+            else
+            {
+                sides.ForEach(side => side.SetOffset(side.meshPos));
+            }
+        }
```

Comment:
Either go collect the list of sides in here, or write a utility method for it.  Don't run on a global list that you have to maintain.  There's several ways to write this, but a 1-line LINQ one is:
```
var sides = part.FindAttachNodes("connect").Where(x => x.attachedPart is Part sp && sp.GetComponent<ProceduralFairingSide>() is ProceduralFairingSide).ToList();
```
Also see comment above about not needing to set `editorOpenOffset.x` directly.